fv3core/stencils/d_sw.py

```diff
@@ -190,92 +333,41 @@ def heat_source_from_vorticity_damping(
     heat_source: FloatField,
     dissipation_estimate: FloatField,
     kinetic_energy_fraction_to_damp: float,
+    dcon_threshold: float,
     calculate_dissipation_estimate: int,
+    do_skeb: bool,
+    damp_vt: float,
 ):
-    """
-    Calculates heat source from vorticity damping implied by energy conservation.
+    from __externals__ import local_ie, local_is, local_je, local_js
 
-    Args:
-        ub (in)
-        vb (in)
-        ut (in)
-        vt (in)
-        u (in)
-        v (in)
-        delp (in)
-        rsin2 (in)
-        cosa_s (in)
-        rdx (in): radius of Earth multiplied by x-direction gridcell width
-        rdy (in): radius of Earth multiplied by y-direction gridcell width
-        heat_source (out): heat source from vorticity damping
-            implied by energy conservation
-        diss_est (out): dissipation estimate, only calculated if
-            calculate_dissipation_estimate is 1
-        kinetic_energy_fraction_to_damp (in): according to its comment in fv_arrays,
-            the fraction of kinetic energy to explicitly damp and convert into heat.
-            TODO: confirm this description is accurate, why is it multiplied
-            by 0.25 below?
-        calculate_dissipation_estimate (in): If 1, calculate dissipation estimate.
-            Equivalent in Fortran model is do_skeb
```

Comment:
would be nice to not lose the docstring for variables still relevant in heat_source_from_vorticity_damping_fxn

Reply:
Added docstring to `heat_source_from_vorticity _damping_fxn`